src/hip/hip_build_utils.cpp

```diff
@@ -129,7 +129,7 @@ boost::filesystem::path HipBuild(boost::optional<TmpDir>& tmp_dir,
     }
     else if(IsHipClangCompiler())
     {
-        params += " -mllvm -amdgpu-enable-global-sgpr-addr";
+        //        params += " -mllvm -amdgpu-enable-global-sgpr-addr";
```

Comment:
This must be reverted.

Reply:
@ltqin Please revert this change in our PR.

If using the flag create issue in this PR, I will fix it in #385

Reply:
@ltqin Yep. Most likely, this is valid change, but it is unrelated to this specific PR.

@asroy wrt #385 -- perhaps we'll need to keep it for older compiler/ROCm versions.

Reply:
@atamazov  done , thanks